tls/s2n_handshake_transcript.c

```diff
@@ -45,8 +43,7 @@ int s2n_conn_update_handshake_hashes(struct s2n_connection *conn, struct s2n_blo
         POSIX_GUARD(s2n_hash_update(&hashes->sha1, data->data, data->size));
     }
 
-    const uint8_t md5_sha1_required = (s2n_handshake_is_hash_required(&conn->handshake, S2N_HASH_MD5) &&
-                                       s2n_handshake_is_hash_required(&conn->handshake, S2N_HASH_SHA1));
+    const uint8_t md5_sha1_required = (s2n_handshake_is_hash_required(&conn->handshake, S2N_HASH_MD5) && s2n_handshake_is_hash_required(&conn->handshake, S2N_HASH_SHA1));
 
```

Comment:
Definitely needs to be wrapped. Why is clang-format undoing all the manual wrapping?

Reply:
Its cause it has opinions about how the wrapping should be done. And when it disagrees with the way we did it, it expresses that by putting stuff on a single line. If I fuss with it I can usually get it to a wrapping we agree on.